src/js/modules/popupRegistration.js

```diff
@@ -54,13 +51,22 @@ function gratitudeOpen() {
 	registerModalGratitude.classList.add("open");
 }
 
-function popupClose(popupContent) {
+function modalClose() {
 	registerModalOverlay.classList.remove("open");
-	popupContent.classList.remove("open");
 	document.removeEventListener("keydown", keyDownHandler);
 	registerModalOverlay.addEventListener("transitionend", enablePageScroll, { once: true });
 }
 
+function popupContentClose(popupContent) {
+	popupContent.classList.remove("open");
+}
+
+function popupsClose() {
+	modalClose();
+	popupContentClose(registerModalContent);
```

Comment:
you use `popupContentClose` only in `popupClose` function, I don't think you need separate function for this code

just write like this
```js
registerModalContent.classList.remove("open");
registerModalGratitude.classList.remove("open");
```

Reply:
Done